lib/cjs/index.d.ts

```diff
@@ -1,22 +1,39 @@
-import esbuild from 'esbuild';
+import esbuild from 'esbuild'
```

Comment:
TODO: I don't know why, but for some reason, when I run `yarn build`, this line (and line 39) are emitted with a semicolon. This also makes the CI fail. Could you add them?

Reply:
eslintrc.json is using semi:never so needed to turn off format on save in vscode.